cli/azd/pkg/apphost/testdata/TestAspireBicepGeneration-resources.bicep.snap

```diff
@@ -105,6 +105,6 @@ output AZURE_CONTAINER_REGISTRY_ENDPOINT string = containerRegistry.properties.l
 output AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID string = managedIdentity.id
 output AZURE_CONTAINER_APPS_ENVIRONMENT_ID string = containerAppEnvironment.id
 output AZURE_CONTAINER_APPS_ENVIRONMENT_DEFAULT_DOMAIN string = containerAppEnvironment.properties.defaultDomain
-output SERVICE_BINDING_POSTGRES_2KV_ENDPOINT string = postgres2kv.properties.vaultUri
-output SERVICE_BINDING_POSTGRES_2KV_NAME string = postgres2kv.name
+output SERVICE_BINDING_KV854251F1_ENDPOINT string = kv854251f1.properties.vaultUri
```

Comment:
An output variable could be used by a user or an external program. Would this be a problem here? It's fine if we want to consider all this implementation detail.

Reply:
It is sort of required in this case, because azd needs to pull secrets from that keyvault later for deploying services.
It is fine b/c the kv access is controlled by rbac.